.github/prepare_assets.sh

```diff
@@ -1,4 +1,23 @@
 #!/usr/bin/env bash
+set -euo pipefail
+
+VERSION="$1"
 
 pip install .
 fhirutil validate --publisher_opts="-tx n/a" --clear_output site_root/ig.ini
```

Comment:
This is a really minor comment and `--clear_output` may not be needed since the line below does `mv -f site_root/output ig`.

Reply:
The clear output flag is used to clear any previous output from the publisher before it runs again. Its to clear out old state so it's still needed in this case.